### PR TITLE
Issues/7350/log update consistency

### LINKED ISSUE #7350: "Log Update"- consistency in terminology
**Is your feature request related to a problem? Please describe.**
Log update is mentioned as "Daily rounds" and "consultation updates" in various places. 

**Describe the solution you'd like**

- [ ] In patient dashboard (https://care.coronasafe.in/facility/0c95c7f0-e1d2-4aff-83fa-933cef60d3a8/patient/7e8fc20f-02da-47f7-8c33-f2d8c7e52dd8/consultation/cb44a173-6fe2-452f-b2d1-6c6e2695411e) 

Screenshot 2024-03-06 at 4 52 50 PM

- [ ]  In patient dashboard (https://care.coronasafe.in/facility/0c95c7f0-e1d2-4aff-83fa-933cef60d3a8/patient/7e8fc20f-02da-47f7-8c33-f2d8c7e52dd8/consultation/cb44a173-6fe2-452f-b2d1-6c6e2695411e) 

Screenshot 2024-03-06 at 4 54 05 PM

Screenshot 2024-03-06 at 4 55 33 PM

- [ ] viewing a saved log update (https://care.coronasafe.in/facility/0c95c7f0-e1d2-4aff-83fa-933cef60d3a8/patient/7e8fc20f-02da-47f7-8c33-f2d8c7e52dd8/consultation/cb44a173-6fe2-452f-b2d1-6c6e2695411e/daily_rounds/2da792c7-7ff8-4b8f-8e68-1397b7848597)
title shows "consultation update"

- [ ] https://care.coronasafe.in/facility/0c95c7f0-e1d2-4aff-83fa-933cef60d3a8/patient/7e8fc20f-02da-47f7-8c33-f2d8c7e52dd8/consultation/cb44a173-6fe2-452f-b2d1-6c6e2695411e/daily-rounds/6469a83e-470e-4fbb-a8ee-51770a42360e Viewing a saved log update, title shows "Consultation update" and bread crumb shows "Daily rounds"

Change all the above to "Log Update"

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by renoseHarsh**
## Proposed Changes

- Fixes #7350 
- In the patient dashboard `src/Components/Facility/ConsultationDetails/ConsultationUpdatesTab.tsx` changed Daily Rounds to Log Updates
Screenshot 2024-08-09 at 12 43 28 AM

- In the patient dashboard  `src/CAREUI/display/Timeline.tsx` added a check to see if the current Timeine is for "daily round" If yes change it to "log update".

- For Notification Title changed the ```NOTIFICATION_EVENTS``` in `src/Common/constants.tsx`
- For Notification Description added a loop to go through incoming API call results in `src/Components/Notifications/NotificationsList.tsx` and change messages from 'consultation' to 'log'
Screenshot 2024-08-09 at 12 19 24 AM

- For viewing saved log updates changed "Consultation Update" to "Log Update" in file `src/Components/CriticalCareRecording/CriticalCare__Index.res`
Screenshot 2024-08-09 at 12 33 14 AM

- For breadcrumbs added mapping for "daily-round" to "Log Update" in `src/Components/Common/Breadcrumbs.tsx`
- For Title changed "Consultation" to "Log"

Screenshot 2024-08-09 at 12 36 09 AM

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

File: src/CAREUI/display/Timeline.tsx [hunk 1]
```diff
@@ -52,6 +52,10 @@ interface TimelineNodeProps {
 
 export const TimelineNode = (props: TimelineNodeProps) => {
   const name = useContext(TimelineContext);
+  let newName: string | null = null;
+  if (props.name === "daily round details Event") {
+    newName = props.name.replace("daily round", "log update");
+  }
```
Comment (rithviknishad) [inline src/CAREUI/display/Timeline.tsx:58]: This is not the right way to solve the problem. 

It should instead be changed in the props passed to this component.

Reusable components such as these should not contain any implementation specific to a particular usage.

File: src/Components/Common/Breadcrumbs.tsx [hunk 2]
```diff
@@ -13,6 +13,7 @@ const MENU_TAGS: { [key: string]: string } = {
   external_results: "External Results",
   users: "Users",
   notice_board: "Notice Board",
+  "daily-rounds": "Log Update",
```
Comment (rithviknishad) [inline src/Components/Common/Breadcrumbs.tsx:16]: Change the URL's path param itself instead

```suggestion
```

File: src/Components/Notifications/NotificationsList.tsx [hunk 3]
```diff
@@ -350,7 +350,26 @@ export default function NotificationsList({
...
+              return {
+                ...notification,
+                message: notification.message.replace(
+                  "Consultation",
+                  "Log Updates",
+                ),
+              };
+            } else {
+              return notification;
+            }
```
Comment (rithviknishad) [inline src/Components/Notifications/NotificationsList.tsx:370]: This doesn't seem to be the right way to do it.

Why is patient consultation created event being renamed to log update? Consultation creation is not relevant to log updates right?

File: src/Components/CriticalCareRecording/CriticalCare__Index.res [hunk 4]
```diff
@@ -113,7 +113,7 @@ let make = (
     <div>
       <div
         className="bg-white px-2 md:px-6 py-5 border-b border-secondary-200 sm:px-6 max-w-5xl mx-auto border mt-4 shadow rounded-lg">
-        <h1 className="text-4xl sm:text-5xl"> {str("Consultation Update")} </h1>
+        <h1 className="text-4xl sm:text-5xl"> {str("Log Update")} </h1>
```
Comment (rithviknishad) [inline src/Components/CriticalCareRecording/CriticalCare__Index.res:116]: Why wrap it in `str(...)`?

```suggestion
        <h1 className="text-4xl sm:text-5xl">Log Update</h1>
```

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Comment by rithviknishad** [comment]
@renoseHarsh It'd be great if you could give a more meaningful title for the PR 😄

**Comment by renoseHarsh** [comment]
Thank you for the feedback. I will make the necessary changes accordingly.

File: src/Components/Notifications/NotificationsList.tsx (hunk 3 above)
Comment (renoseHarsh) [reply src/Components/Notifications/NotificationsList.tsx:370]: Can you clarify if the way I am modifying data is wrong or is it just the part that I have 'PATIENT_CONSULTATION_CREATED' in toChangeId wrong?

Comment (rithviknishad) [reply src/Components/Notifications/NotificationsList.tsx:370]: Consultation Create notification message need not be replaced with log updates right?

**Pull request closed without merging**